### PR TITLE
Add more fields to escalation policy section

### PR DESCRIPTION
**Pull request by violetyao**
Jira tickets
- https://issues.redhat.com/browse/APPSRE-2879
- https://issues.redhat.com/browse/APPSRE-3019 

Displays JiraBoard, SlackUserGroup, nextEscalationPolicy, Email, and PagerDuty, if they are not null. 
Escalation Policy schema: https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/schemas/app-sre/escalation-policy-1.yml

### CODE REVIEW

File: src/pages/elements/Service.js [hunk 1]
```diff
@@ -21,29 +21,84 @@ const linkFormat = url => value => (
 const emptyFormat = value => value || '-';
 const booleanFormat = (t, f) => value => (value ? t : f);
 
+// link to window.DATA_DIR_URL path
```
Comment (BumbleFeng) [inline src/pages/elements/Service.js:24]: I think we should put the following part in components folder.

File: src/pages/elements/Service.js [hunk 2]
```diff
@@ -21,29 +21,84 @@ const linkFormat = url => value => (
...
-              {app.escalationPolicy.name}
-            </a>
+            <LinkPath item={app.escalationPolicy}/>
           ],
           [
             'Description',
             <pre>{app.escalationPolicy.description}</pre>
+          ],
+          [
+            'Next Escalation Policy',
```
Comment (BumbleFeng) [inline src/pages/elements/Service.js:82]: Next Escalation Policy should be the last on the list

File: src/pages/elements/Service.js (hunk 1 above)
Comment (violetyao) [reply src/pages/elements/Service.js]: Thanks for your feedback! I've moved them to NonEmptyDefinition.js/ NamePathList.js under components.

File: src/pages/elements/Service.js (hunk 2 above)
Comment (violetyao) [reply src/pages/elements/Service.js]: Done!

**Review by BumbleFeng — APPROVED** [review]

**Pull request merged**